scrapers/vixenNetwork.py

```diff
@@ -0,0 +1,90 @@
+import json
+import os
+import re
+import requests
+import sys
+from urllib.parse import urlparse
+
+
+def log(*s):
+    print(*s, file=sys.stderr)
+
+
+def get_from_url(url_to_parse):
+    p = urlparse(url_to_parse)
+    return p.netloc, p.netloc[4:-4], p.path[1:]
```

Comment:
This is not actually that safe
In the yaml file we match vixen.com/ so in theory a user could have there `vixen.com/this-whole-time`
Using something like `ltdextract` to get the domain name would be probably better.

Reply:
Yup! You're right! There are four ways the URL could be inserted and two of them would end up broken. `tldextract` is something that users would have to install so I'm not doing that, especially when there's a simple answer available with RegEx:

```py
m = re.match(r'(?:https?:\/\/)?(?:www\.)?((\w+)\.com)\/([a-z-]+)', url_to_parse)
return  m.groups()
```